### ISSUE TITLE
Training a model and generating a prediction

### ISSUE THREAD

**Issue by jmm5491** [qlemaire22/speech-music-detection#10]
I want to thank you for making this significant contribution to the field.  The project was extremely well done, and I have learned a lot about how to structure a large python project from reading your code.

I have attempted to train a model using the 'all_quality' config in experiments.json using a subset of the datasets (musan, gtzan, muspeak).  The training was aborted after 28/50 epochs due to a tf/keras bug, and I was saving the model after each epoch.  The last model saved had these results:

1214/1214 [==============================] - 557s 457ms/step - loss: 0.1321 - binary_accuracy: 0.9514 - categorical_accuracy: 0.8334 - val_loss: 0.1099 - val_binary_accuracy: 0.9621 - val_categorical_accuracy: 0.9421

I thought I might be able to use that model to make a prediction on a 30 minute wav file (mostly music with 4 segments of speech of about 3 minutes each).  The output of predict.py labeled the entire 30 minutes as speech, so I think I'm doing something wrong.  I wasn't sure what to put for the --mean_path and --std_path.  I just see mean.npy and var.npy files in the filelists_* directories of the datasets.  Are these supposed to be combined in some way and the result passed to predict.py?

I would be grateful for any advice you can give about training and running predictions.

Merci

**Comment by adhvaidh1** [comment]
Hello, 
Hope you're doing well. I was wondering if you paste your experiments.json file, I keep getting the "Some audio files could not enter in the batch composition. Excluded files:" error, and my val_acc keeps decreasing, I'm using a subset of musan and esc-50 for (noise, music). Thanks.
